frontend/javascripts/test/puppeteer/dataset_rendering.screenshot.ts

```diff
@@ -106,19 +106,38 @@ const datasetConfigOverrides: Record<string, PartialDatasetConfiguration> = {
   },
 };
 
+const datasetNameToId: Record<string, string> = {};
+datasetNames.map(async (datasetName) => {
```

Comment:
I think this test does not do what its description and assertion state. Instead it fetches the ids for all datasets. Doing that as part of a test seems hacky. Can't you use [test.before](https://github.com/avajs/ava/blob/main/docs/01-writing-tests.md#:~:text=or%20teardown%20code.-,test.before(),-registers%20a%20hook) for that?